### PR TITLE
Localized all Case Studies in hi

### PR DESCRIPTION
**Pull request by kumarankit999**

### CODE REVIEW

**Comment by kumarankit999** [comment]
Issue resolved #43271

**Comment by utkarsh-singh1** [comment]
Netlify build failed due to this error - `Error: Error building site: "/opt/build/repo/content/hi/case-studies/appdirect/index.html:11:1": failed to unmarshal YAML: yaml: line 11: could not find expected ":"`

**Comment by kumarankit999** [comment]
How can I resolve it???

**Comment by kumarankit999** [comment]
@utkarsh-singh1

**Review by aj11anuj — COMMENTED** [review]
🙌 Thanks for your effort and time, that you have putted to localize these 4 pages with proper images. It's advisable and feasible to break this huge PR into small-small 4 PR's for a smooth feedback process.

**Comment by kumarankit999** [comment]
Should I make 4 PR's for this

**Comment by Gauravpadam** [comment]
@kumarankit999 It's a better practice to, But it's not necessary

> Should I make 4 PR's for this

File: content/hi/case-studies/appdirect/index.html [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+title: ऐपडायरेक्ट केस स्टडी
+linkTitle: AppDirect
+case_study_styles: true
+cid: caseStudies
+logo: appdirect_featured_logo.png
+featured: true
+weight: 4
+quote: >
+हमने सही समय पर सही निर्णय लिये। कुबेरनेट्स और क्लाउड नेटिव प्रौद्योगिकियों को अब वास्तविक पारिस्थितिकी तंत्र के रूप में देखा जाता है।
```
Comment (Gauravpadam) [inline content/hi/case-studies/appdirect/index.html:10]: This seems like a linting error, Try adding a few whitespaces before this text

File: content/hi/case-studies/babylon/index.html [hunk 2]
```diff
@@ -0,0 +1,84 @@
+---
+title: बेबीलोन केस स्टडी
+linkTitle: Babylon
+case_study_styles: true
+cid: caseStudies
+logo: babylon_featured_logo.svg
+featured: true
+weight: 1
+quote: >
+कुबेरनेट्स मशीन लर्निंग के लिए एक बेहतरीन मंच है क्योंकि यह आपके लिए आवश्यक सभी शेड्यूलिंग और स्केलेबिलिटी के साथ आता है।
```
Comment (Gauravpadam) [inline content/hi/case-studies/babylon/index.html:10]: Same, Indent this with some whitespaces

File: content/hi/case-studies/booking-com/index.html [hunk 3]
```diff
@@ -0,0 +1,86 @@
+---
+title: Booking.com Case Study
+linkTitle: Booking.com
+case_study_styles: true
+cid: caseStudies
+logo: booking.com_featured_logo.png
+featured: true
+weight: 3
+quote: >
+हमने महसूस किया कि कुबेरनेट्स की क्षमता का पूरी तरह से उपयोग करने के लिए हमें इसे बेहतर तरीके से सीखने की जरूरत है। उस समय, हमने अपना स्वयं का कुबेरनेट्स प्लेटफ़ॉर्म बनाने का निर्णय लिया।
```
Comment (Gauravpadam) [inline content/hi/case-studies/booking-com/index.html:10]: Same goes here

File: content/hi/case-studies/booz-allen/index.html [hunk 4]
```diff
@@ -0,0 +1,80 @@
+---
+title: बूज़ एलन केस स्टडी
+linkTitle: बूज़ एलन हैमिल्टन
+case_study_styles: true
+cid: caseStudies
+logo: booz-allen-featured-logo.svg
+featured: true
+weight: 2
+quote: >
+कुबेरनेट्स हमारे लिए एक बेहतरीन समाधान है। यह हमें अपने ग्राहकों की मांगों पर तेजी से अमल करने की अनुमति देता है।
```
Comment (Gauravpadam) [inline content/hi/case-studies/booz-allen/index.html:10]: And here

**Review by Gauravpadam — COMMENTED** [review]
Thanks for the PR,

A technical review for the front matter from my end,

@kumarankit999 Try these and then make a commit, Should fix the build

**Review by Gauravpadam — COMMENTED** [review]
A few linting errors, YAML acts up sometimes!

This should fix the build

**Comment by kumarankit999** [comment]
Should I recreate this PR, or create 4 PR's for this

**Comment by utkarsh-singh1** [comment]
Hi @kumarankit999, it would be better to raise another PR on this same branch. On the same branch, commit your changes locally and raise PRs here.

File: content/hi/case-studies/appdirect/index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (utkarsh-singh1) [reply content/hi/case-studies/appdirect/index.html:10]: YAML is sensitive to indents as compare to JSON, that's why this seems to be an issue as it considers `line 10` content as a key.

**Comment by kumarankit999** [comment]
> Hi @kumarankit999, it would be better to raise another PR on this same branch. On the same branch, commit your changes locally and raise PRs here.

Should I Create 4 PR's

**Comment by Gauravpadam** [comment]
For now, Fix this PR. Create separate ones if an approver says so

**Comment by utkarsh-singh1** [comment]
> Should I Create 4 PR's

@kumarankit999, Raising 4 PRs for a same issue is not feasible, as each PR needs to go through [Netlify](https://www.netlify.com) build tool, you can check the log of this PR under [checks](https://github.com/kubernetes/website/pull/43516/checks), means consumption of resources will also increase, Always try to avoid multiple PRs for single issue unless, the changes required in multiple languages as each language has its own guidelines to improve their doc.

**Comment by kumarankit999** [comment]
OK, thanks you all for your suggestions. I will create another PR for this.

**Pull request closed without merging**